### DISCUSSION TITLE
when use 	protected $cacheCooldownSeconds = 300; // 5 m not expire

### DISCUSSION THREAD

**Discussion by vahidalvandi** [GeneaLabs/laravel-model-caching#489] [General]
i set  $cacheCooldownSeconds = 300 but after edit database manually  after 5 minute cache not rest

**Comment by mikebronner** [comment]
@vahidalvandi Please provide more information:

### Eloquent Query

Please provide the complete eloquent query that caused the bug, for example:

$model->with('otherModel')->get();
Stack Trace The full stack trace from your log file.

### Environment

PHP: [e.g. 7.1.0]
OS: [e.g. Ubuntu 18.04]
Laravel: [e.g. 5.6.15]
Model Caching: [e.g. 0.2.61]

### Additional Context

Add any other context about the problem here.

**Comment by liuwei73** [comment]
try this

`public $cacheCooldownSeconds = 300;`
